Collections DB.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+$db = new PDO('mysql:host=db; dbname=collections', 'root', 'password');
+
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```

Comment:
I would suggest popping these two lines into a function.

That way, when it comes to adding more pages to your application, you don't need to copy and paste these lines to make a new connection, you can just call your own function which gives you a database connection